lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -1,37 +1,40 @@
-;(function (define, undefined) {
+;(function(define, undefined) {
     'use strict';
     define([
-        'gettext', 'jquery', 'underscore', 'backbone', 'js/views/fields', 'js/views/image_field', 'backbone-super'
-    ], function (gettext, $, _, Backbone, FieldViews, ImageFieldView) {
+        'gettext', 'jquery', 'underscore', 'backbone', 'js/views/fields', 'js/views/image_field',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'backbone-super'
+    ], function(gettext, $, _, Backbone, FieldViews, ImageFieldView, HtmlUtils) {
 
         var LearnerProfileFieldViews = {};
 
         LearnerProfileFieldViews.AccountPrivacyFieldView = FieldViews.DropdownFieldView.extend({
 
-            render: function () {
+            render: function() {
                 this._super();
                 this.showNotificationMessage();
                 this.updateFieldValue();
                 return this;
             },
 
-            showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+            showNotificationMessage: function() {
+                var accountSettingsPageUrl = this.options.accountSettingsPageUrl,
+                    accountSettingsLink = HtmlUtils.HTML(
+                        '<a href="' + accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>'
+                    );
                 if (this.profileIsPrivate) {
-                    this._super(interpolate_text(
-                        gettext("You must specify your birth year before you can share your full profile. To specify your birth year, go to the {account_settings_page_link}"),
+                    this._super(HtmlUtils.interpolateHtml(
+                        gettext('You must specify your birth year before you can share your full profile. To specify your birth year, go to the {account_settings_page_link}'),  // jshint ignore:line
                         {'account_settings_page_link': accountSettingsLink}
```

Comment:
1. Rename `accountSettingsLink` to `accountSettingsLinkHtml`.
2. Even though it is redundant, I think I might code this with `HtmlUtils.HTML(accountSettingsLink)`.